lib/datasource/conda/index.spec.ts

```diff
@@ -0,0 +1,65 @@
+import { getPkgReleases } from '..';
+import { Fixtures } from '../../../test/fixtures';
+import * as httpMock from '../../../test/http-mock';
+import { EXTERNAL_HOST_ERROR } from '../../constants/error-messages';
+import { datasource, defaultRegistryUrl } from './common';
+
+const depName = 'main/pytest';
+const depUrl = `/${depName}`;
+
+describe('datasource/conda/index', () => {
+  describe('getReleases', () => {
+    it('throws for error', async () => {
+      httpMock.scope(defaultRegistryUrl).get(depUrl).replyWithError('error');
+      await expect(
+        getPkgReleases({
+          datasource,
+          depName,
+        })
+      ).rejects.toThrow(EXTERNAL_HOST_ERROR);
+    });
+
+    it('returns null for 404', async () => {
+      httpMock.scope(defaultRegistryUrl).get(depUrl).reply(404);
+      expect(
+        await getPkgReleases({
+          datasource,
+          depName,
+        })
+      ).toBeNull();
+    });
+
+    it('returns null for empty result', async () => {
+      httpMock.scope(defaultRegistryUrl).get(depUrl).reply(200, {});
+      expect(
+        await getPkgReleases({
+          datasource,
+          depName,
+        })
+      ).toBeNull();
+    });
+
+    it('throws for 5xx', async () => {
+      httpMock.scope(defaultRegistryUrl).get(depUrl).reply(502);
+      await expect(
+        getPkgReleases({
+          datasource,
+          depName,
+        })
+      ).rejects.toThrow(EXTERNAL_HOST_ERROR);
+    });
+
+    it('processes real data', async () => {
+      httpMock
+        .scope(defaultRegistryUrl)
```

Comment:
The url needs ending slash to get proper scope. 😉

Reply:
Why?

nock itself doesn't need that as per its documentation (https://github.com/nock/nock#usage) and I don't see [the `scope` code](https://github.com/renovatebot/renovate/blob/b6c374f16698b1a20c01cdeb1de353634a698dc9/test/http-mock.ts#L60-L82) treating it differently.

However, it looks to me like nock strips out the trailing slash anyways, as the following spec **with** a trailing slash at the `baseUrl` still is missing a slash.

```ts
    it('test with literals', async () => {
      httpMock
        .scope('https://api.anaconda.org/package/')
        .get('main/pytest')
        .reply(200, Fixtures.get('pytest.json'));
      const res = await getPkgReleases({
        datasource,
        depName,
      });
      expect(res).toMatchSnapshot();
      expect(res?.releases).toHaveLength(94);
    });
```

Will result in

```
  ● datasource/conda/index › getReleases › test with literals

    Pending mocks!
     * GET https://api.anaconda.org:443/packagemain/pytest
```

Reply:
I'll resolve this as I updated the config and it makes sense to do so, but if you can provide more insight, that would be great @viceice.